blenderproc/api/utility/__init__.py

```diff
@@ -1,3 +1,7 @@
-from blenderproc.python.utility.Utility import resolve_path, num_frames, resolve_resource, set_keyframe_render_interval, reset_keyframes
+from blenderproc.python.utility.Utility import resolve_path, num_frames, resolve_resource, set_keyframe_render_interval, reset_keyframes, Utility
 from blenderproc.python.utility.LabelIdMapping import LabelIdMapping
 from blenderproc.python.utility.PatternUtility import generate_random_pattern_img
+
+# extract them from the Utility class for the public API
+UndoAfterExecution = Utility.UndoAfterExecution
+BlockStopWatch = Utility.BlockStopWatch
```

Comment:
Usually we do not do that, because this destroys the link to the actual class which has documentation (useful when you perform ctrl+click on `bproc.utility.UndoAfterExecution` in you main.py script).

So, the better way would be:
1. Move UndoAfterExecution and BlockStopWatch out of the Utility scope
2. Then you can directly import it here via `from blenderproc.python.utility.Utility import [...],UndoAfterExecution`

This also has the advantage that we are not importing the Utility class into the api